.gitattributes

```diff
@@ -5,5 +5,6 @@
 phpunit.xml.dist       export-ignore
 phpcs.xml.dist         export-ignore
 psalm.xml.dist         export-ignore
-/Resources/doc         export-ignore
-/Tests                 export-ignore
+/docs                  export-ignore
+/tests                 export-ignore
+.symfony.bundle.yaml   export-ignore
```

Comment:
is it fine to exclude this? probably its not used with releases, or?

Reply:
cc @javiereguiluz

Reply:
Yes, we can exclude it. In symfony.com we use GitHub API to get the file contents directly from the default branch of the repository. Thanks.